session/bootstrap.go

```diff
@@ -209,6 +209,20 @@ const (
 		feedback blob NOT NULL,
 		index hist(table_id, is_index, hist_id)
 	);`
+
+	// CreateBindInfoTable stores the sql bind info which is used to update globalBindCache.
+	CreateBindInfoTable = `CREATE TABLE IF NOT EXISTS mysql.bind_info (
+		original_sql varchar(2048) NOT NULL  ,
+      	bind_sql varchar(2048) NOT NULL ,
+      	default_db varchar(1024)  NOT NULL,
+		status int(10) NOT NULL DEFAULT 1,
+		create_time timestamp NOT NULL,
+		update_time timestamp NOT NULL,
+		charset varchar(100) NOT NULL,
+		collation varchar(100) NOT NULL,
+ 		UNIQUE KEY sql_db (original_sql, default_db),
```

Comment:
for one `(original_sql, default_db)` pair, only one `charset` and `collation` is allowed to be used to create a SQL Binding?

Reply:
the `charset` and `collation` is using for parse the origin sql and bind sql.